x-pack/plugins/security/public/nav_control/nav_control_component.scss

```diff
@@ -0,0 +1,9 @@
+.securityNavControlComponent__userMenu {
+  .euiPopoverTitle {
+    text-transform: none;
+  }
+
+  .securityNavControlComponent__logoutLink {
+    border-top: $euiBorderThin;
+  }
```

Comment:
I'd remove this or only apply it if cloud links exist. It looks a bit odd when there's only two items.

Reply:
I ended up using the separator menu item instead of relying on CSS to add that divider. https://github.com/elastic/kibana/pull/66825/files#diff-244d6513d4e0e5f7effeaa007127777d567372db67bf0e3ae54bbea6481bd2a8R160